kars/0003-gpu-sharing/README.md

```diff
@@ -0,0 +1,40 @@
+# KAR-0003: GPU Sharing
+
+## Description
+
+If accelerator supports GPU sharing, provide well-defined mechanisms for GPU sharing to improve utilization for workloads that do not require a full dedicated GPU. If hardware-level partitioning is supported, these fractional GPU resources should be exposed as distinct schedulable resources via DRA once it’s supported. If software-based sharing (e.g. time-slicing) is supported, device plugin should be used to allow oversubscription of GPUs, with a forward-looking path to DRA.
+
+## Motivation
+
+AI/ML workloads on Kubernetes need GPU sharing because it dramatically improves GPU utilization. Most inference and fine-tuning jobs don’t need an entire modern GPU. Partitioning lets multiple workloads run concurrently without wasting compute. It also reduces cost by allowing teams to pack more jobs onto the same hardware, and enables resource isolation so different users or namespaces can safely share the same GPU without interfering with each other. GPU sharing also provides scheduling flexibility, letting Kubernetes treat GPUs more like fine-grained, composable resources rather than monolithic devices.
+
+## Graduation Criteria
+
+**SHOULD**
+- [x] Describe how users can test it for self-attestation with scripts, documentation, etc
+- [ ] Starting v1.37, new SHOULDs must include proposed automated tests in the automated tests section below
+
+**MUST**
+- [ ] Starting v1.37, new MUSTs must include automated tests that have been added to the AI conformance test suite
+- [ ] Demonstrate at least two real-world usage of SHOULD before graduating to MUST
+- [ ] Kubernetes core APIs must be GA
+
+## Test Plan
+
+### How We Might Test It
+
+We should be able to use the verifiable mechanism provided by the platform to ensure that a single physical GPU can be exposed as multiple allocatable GPU resources and scheduled by Kubernetes. Create functional tests that verify multiple workloads requesting multiple logical GPUs can be scheduled and are running on the same physical GPU. Another test to ensure over allocating logical GPUs is blocked. 
```

Comment:
Would a platform meet this compliance requirement if say the platform allows GPU sharing but only via the Nvidia GPU operator, that would need to be installed by the end user?

Reply:
As discussed in the 12/4/2025 community call, a platform's "verifiable mechanism" can be a document describing the steps users make to meet this requirement. When submitting for certification, the proof can include these steps and the automated tests can run _AFTER_ these steps are performed.

Reply:
Thanks, so sounds like the answer to that question is "yes"? If the platform documents the steps to skip managed installation of the GPU dependencies, and instead use the Nvidia GPU operator for a particular GPU sharing feature then it should be compliant (provided the automated tests pass), then the platform is compliant right?